upgrade_tests/helpers/constants.py

```diff
@@ -7,7 +7,7 @@
 FAKE_REPO_ZOO3 = 'http://inecas.fedorapeople.org/fakerepos/zoo3/'
 FROM_VERSION = os.environ.get('FROM_VERSION')
 TO_VERSION = os.environ.get('TO_VERSION')
-SUPPORTED_SAT_VERSIONS = ['6.2', '6.3', '6.4', '6.5', '6.6', '6.7', '6.8', '6.9']
+SUPPORTED_SAT_VERSIONS = ['6.7', '6.8', '6.9']
```

Comment:
@swadeley I believe 6.10 should be appended here.

Reply:
Hi @Gauravtalreja1 , I have added that.

Reply:
@Gauravtalreja1 I think @devendra104 meant to say this section is obsolete due to this: https://github.com/SatelliteQE/robottelo/blob/master/conf/upgrade.yaml.template